ioreg/src/builder/accessors.rs

```diff
@@ -122,6 +126,29 @@ fn build_get_fn(cx: &ExtCtxt, path: &Vec<String>, reg: &node::Reg)
   item.unwrap()
 }
 
+fn build_ignoring_state_setter_fn(cx: &ExtCtxt, path: &Vec<String>, reg: &node::Reg)
+                -> P<ast::Item>
+{
+  let reg_ty: P<ast::Ty> =
+    cx.ty_ident(DUMMY_SP, utils::path_ident(cx, path));
```

Comment:
to avoid hitting lint errors after #298 is done, should do

```diff
-DUMMY_SP
+reg.name.span
```

Because that's where the name of this register is derived from.